templates/terraform/examples/monitoring_slo_appengine.tf.erb

```diff
@@ -0,0 +1,19 @@
+data "google_monitoring_app_engine_service" "default" {
+  module_id = "default"
+}
+
+resource "google_monitoring_slo" "<%= ctx[:primary_resource_id] -%>" {
```

Comment:
Can you also add a super basic test that sets very few of the fields, to ensure we're handling defaults correctly?

Reply:
I edited the resource monitoring slo test to give minimal fields - goal is required, one of calendar_period/rolling_period is required, and one of the SLIs (only basic is supported in this PR for now) is required. The only truly optional fields are slo_id and display name.